sdk-java/src/main/java/org/wsecm/Application.java

```diff
@@ -0,0 +1,16 @@
+package org.wsecm;
+
+import org.wsecm.service.WatchService;
+import org.wsecm.task.FetchSecretsTaskFactory;
+import org.wsecm.task.TaskFactory;
+
+import static org.wsecm.HealthCheckServer.startServer;
+
+public class Application {
+    public static void main(String[] args) {
+        startServer();
+        WatchService watchService = WatchService.getInstance();
+        TaskFactory taskFactory = new FetchSecretsTaskFactory();
+        watchService.scheduleTask(taskFactory);
```

Comment:
I am assuming these are just to test the functionality, and the `main()` function will not be a part of the SDK. — we are providing a library; not an executable.

We’ll have two functions:

* `Watch()`
* `Fetch()` 

We don’t need a health check server.

--

This will not be a Pod; it will be a library where others can build their own pods.